tests/foreman/cli/test_domain.py

```diff
@@ -98,9 +98,6 @@ def valid_delete_params():
 class TestDomain(CLITestCase):
     """Domain CLI tests"""
 
-    factory = make_domain
```

Comment:
```py:class reference target not found: Domain```
These lines were missed when we decided to drop MetaCLITests. No more need in these class attributes.